### DISCUSSION TITLE
New task icons showing up on our workload home screen

### DISCUSSION THREAD

**Discussion by box-turtle** [microsoft/Microsoft-Fabric-workload-development-sample#71] [Help]
Before when we navigated to our workload home we saw this:

![image](https://github.com/user-attachments/assets/de6cacdd-735e-4b82-883b-c32d31ac655a)

But now we see a new set of icons that pushes our items off to the right of the screen. 

![image](https://github.com/user-attachments/assets/7b37dc98-e5d9-4ee7-9f59-d646f49fc5d2)

Can we control this via configuration? We'd rather not see those on our workload home.

**Comment by IdanGoldfarb1989** [comment]
This is part of the general changes made to the Fabric home page across all experiences. Unfortunately, workloads can't control it.
